src/static/js/modules/youtube.js

```diff
@@ -7,8 +7,6 @@
 
```

Comment:
Good call. We should make that a future task so we can get this merged and keep moving forward.